### PR TITLE
Added option to delete services related to replication controller while deleting RC

### PR DESCRIPTION
**Pull request by floreks**
Related to #323 

I've added option to delete services related to replication controller with some tooltip.

## Concept

Let's take simple label selector `{app: test}`

I'm checking if this selector targets only 1 replication controller. This is the most important check, because if there are more replication controllers targeted by this selector then we can not delete any related services.

If there is only 1 replication controller then services targeted by this selector can be deleted.

zrzut ekranu z 2016-02-03 15-27-02

Is tooltip message clear enough?

### CODE REVIEW

File: src/app/frontend/replicationcontrollerdetail/deletereplicationcontroller.html [hunk 1]
```diff
@@ -21,6 +21,14 @@ <h4 class="md-title">Delete Replication Controller</h4>
       Delete replication controller {{::ctrl.replicationController}} in namespace {{::ctrl.namespace}}.<br>
       Pods managed by the replication controller will be also deleted.
     </div>
+    <md-checkbox ng-model="ctrl.deleteServices" class="kd-deletedialog-services-checkbox">
+      Delete related services
+      <md-icon class="material-icons kd-deletedialog-info-icon">
+        info
+        <md-tooltip>Services with label selector matching only this replication controller
+          will be deleted</md-tooltip>
```
Comment (floreks) [inline src/app/frontend/replicationcontrollerdetail/deletereplicationcontroller.html]: Are tooltip and checkbox message ok?

**Comment by floreks** [comment]
@maciaszczykm @bryk PTAL

**Comment by maciaszczykm** [comment]
LGTM

File: src/app/backend/apihandler.go [hunk 2]
```diff
@@ -297,11 +297,24 @@ func (apiHandler *ApiHandler) handleUpdateReplicasCount(
...
 
 	namespace := request.PathParameter("namespace")
 	replicationController := request.PathParameter("replicationController")
+	deleteServices, err := strconv.ParseBool(request.QueryParameter("deleteServices"))
+	if err != nil {
+		handleInternalError(response, err)
+		return
+	}
+
+	if deleteServices {
```
Comment (bryk) [inline src/app/backend/apihandler.go]: Apihandlers should be logic-less. Can you move this to DeleteReplicationController and pass the boolean to it? The idea is to have zero logic in this file and just pass arguments to implementation functions.

Comment (floreks) [inline src/app/backend/apihandler.go]: Sure.

**Comment by bryk** [comment]
Looks really nice. One comment. PTAL

**Comment by floreks** [comment]
Moves logic away from api handler. Is function name still ok?

PTAL

**Comment by bryk** [comment]
I think not. How about naming it DeleteReplicationController and then just documenting its behavior in docs?

**Comment by floreks** [comment]
Ye, I think that's the best option

**Comment by bryk** [comment]
Comment PTAL when you're ready.

**Comment by floreks** [comment]
PTAL

**Comment by bryk** [comment]
LGTM :)

**Pull request merged**